### ISSUE TITLE
Support of Amazon EKS

### ISSUE THREAD

**Issue by shahidhk** [hasura/kubeformation#10]
EKS is in closed preview. We have the implementation internally, but cannot open souce until EKS becomes generally available. Will update once that happens.

**Comment by mahmoudhossam** [comment]
@shahidhk EKS is now [generally available].

[generally available]: https://aws.amazon.com/blogs/aws/amazon-eks-now-generally-available/

**Comment by shahidhk** [comment]
Yes! But they changed some stuff for the GA. Need to spend some time to figure it out. Any help is appreciated!
